lib/load_aws.rb

```diff
@@ -2,14 +2,14 @@
 
 def generate_aws_yml_file
   sample_aws = {"aws"=>{"access_key_id"=>"", "secret_access_key"=>"", "region"=>""}}
-  File.open(File.join(Dir.pwd, 'aws.yml'), 'w') do |file|
+  File.open(File.join(Dir.pwd, 'config/aws.yml'), 'w') do |file|
```

Comment:
How about having the path to the aws.yml file in a constant and then use it everywhere?